File: app/src/main/java/br/pedroso/upcomingmovies/movieslist/MoviesListView.java
package br.pedroso.upcomingmovies.movieslist;

import java.util.List;

import br.pedroso.upcomingmovies.domain.Movie;

public interface MoviesListView {
    void renderMoviesList(List<Movie> moviesList);

    void startMovieDetailsActivity(Integer id);

    void cleanMoviesList();
}

File: app/src/main/kotlin/br/pedroso/upcomingmovies/movieslist/MoviesListView.kt
package br.pedroso.upcomingmovies.movieslist

import br.pedroso.upcomingmovies.domain.Movie

interface MoviesListView {
    fun renderMoviesList(moviesList: List<Movie>)

    fun startMovieDetailsActivity(movieId: Int)

    fun cleanMoviesList()
}
